js-green-licenses.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packageWhitelist": [
+    "log-driver"  // https://github.com/googleapis/nodejs-common/issues/16
```

Comment:
The comment makes the JSON invalid :(

Reply:
`jsgl` allows comments here, by stripping them out before parsing :)